core/src/main/java/io/grpc/internal/ClientTransport.java

```diff
@@ -136,18 +136,18 @@ ClientStream newStream(MethodDescriptor<?, ?> method,
 
     /**
      * Invoked when a ping is acknowledged. The given argument is the round-trip time of the ping,
-     * in microseconds.
+     * in nanoseconds.
      *
-     * @param roundTripTimeMicros the round-trip duration between the ping being sent and the
+     * @param roundTripTimeNanos the round-trip duration between the ping being sent and the
      *     acknowledgement received
      */
-    void pingAcknowledged(long roundTripTimeMicros);
+    void onSuccess(long roundTripTimeNanos);
```

Comment:
Oh. I guess this was detect that those APIs needed to convert from micros to nanos. I guess that's fine, although I would have probably preferred to leave this API as-is since it is primarily there for external use (we don't use it at all internally).